static/docs/user-guide/install/index.md

```diff
@@ -0,0 +1,62 @@
+# Install
+
```

Comment:
A very similar page exists in https://dvc-org-pr-695.herokuapp.com/doc/get-started/install which should probably be rewritten much shorter and just link to this guide for more details?

Reply:
> A very similar page exists in https://dvc-org-pr-695.herokuapp.com/doc/get-started/install which should probably be rewritten much shorter and just link to this guide for more details?

Let's postpone this for another PR after this one.

Reply:
I don't think we should merge a PR that duplicates a bunch of content. If you'd like you can remove all these semi-duplicated pages from the PR so we can get it done, and add them in a next one where the old ones can be deleted at the same time, then we'll just review that the migration is good 🙂

Reply:
@shcheklein seems to think this is OK though, per https://github.com/iterative/dvc.org/pull/695#pullrequestreview-302319707. I vote against it, but you're 
the merge master Ivan 😎  Your call.

Reply:
@jorgeorpinel Simplifying `/get-started/install` is not a simple task. After simplification you may need to merge it with `/get-started/initialize`. Then you have to change all the links that refer to `/get-started/install` to link to the new installation page. This may potentially cause any conflicts with any other PRs.
Let's keep PRs small, focused and quick, to avoid potential conflicts.

Reply:
Well. I think you can basically remove the get-started/install and just add a link on the top of initialize, so maybe it's not that complicated. But as you guys choose.

Let's leave these unresolved though, and/or open a new issue detailing all these duplication problems and linking back to these discussions?

Reply:
I think @dashohoxha already opened a new issue #701. May be it worth putting the link to this discussion in that ticket.

Reply:
Linked